en/flight_controller/kakutef7.md

```diff
@@ -0,0 +1,158 @@
+# Kakute F7
+
+The *Kakute F7* from Holybro is a flight controller board designed for racers. 
+
+<img src="../../assets/flight_controller/kakutef7/board.jpg" width="400px" title="Kakute F7" />
+
+
+## Key Features
+
+* Main System-on-Chip: [STM32F745VGT6](https://www.st.com/en/microcontrollers-microprocessors/stm32f745vg.html)
+  * CPU: 216 MHz ARM Cortex M7 with single-precision FPU
+  * RAM: 320 KB SRAM
+  * FLASH: 1 MB
+* Standard racer form factor: 36x36 mm with standard 30.5 mm hole pattern
+* ICM20689 Accel / Gyro (Soft-mounted)
+* BMP280 Baro
+* microSD (for logging)
+* 6 UARTs
+* 1 I2C bus
+* 6 PWM outputs
+* built-in OSD chip (AB7456 via SPI)
+
+
+## Where to Buy
+
+The board can be bought for example from one of the following shops:
+- [Holybro](https://shop.holybro.com/kakute-f7_p1104.html) 
+- [getfpv](https://www.getfpv.com/holybro-kakute-f7-tekko32-f3-metal-65a-4-in-1-esc-combo.html)
+
+> **Tip** The Kakute F7 fits with the Tekko32 4-in-1 ESC.
```

Comment:
By "fits" do you mean "is designed to work with?" or "comes with"? If the former, might be worth saying that.

Not a general rule, but often it is easier to parse product names visually if they are in italic. So you might if you can be bothered markup everywhere product as *Kakute F7*.